modules/farm_fd2/src/entrypoints/transplanting/lib.submit.unit.cy.js

```diff
@@ -0,0 +1,410 @@
+import { lib } from './lib.js';
+import * as farmosUtil from '@libs/farmosUtil/farmosUtil';
+
+describe('Submission using the transplanting lib.', () => {
+  let form = {
```

Comment:
Consider creating a new Tray Seeding to be used for this test so it is not dependent upon the current database state.  Can we `import lib.js` from `tray_seeding`?

Add `'@fd2': fileURLToPath(new URL('entry_points/', import.meta.url)),` to the `vite.config.js` and `vite.config.e2e.js` files in `farm_fd2`.

Then import with something like:

`import {lib as traySeedingLib} from @fd2/tray_seeding/lib.js`